### PR TITLE
Added breakdown by reviewer in dashboard

### PR DESCRIPTION
**Pull request by Benson272**
### Administrative Info
Monday Board ID: 1181559666
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added a table in the dashboard to show how many applications each reviewer has gone through

### Testing
How did you confirm your changes worked? 
- Created applications and checked table to see if it accurately reflects reviews

### Confirmation of Change 
Table when there are no applications to review
image

One application submitted (assigned to PM)
image

Two applications in database (one assigned to PM, one assigned to President)
image

president reviewed application (note president now has one accepted and pm now has one in Phone phase)
image

pm reviewed person in phone stage and rejected the person (not pm  now has one rejection)
image

### CODE REVIEW

**Comment by justinyaodu** [comment]
Hi Benson, could you please run `npm run lint-fix` to address the code style issues? It seems like Prettier wasn't run before the commit.

**Comment by justinyaodu** [comment]
Also, there seem to be a large number of unknown files that were created under `backend/` - could you please delete those? (I think the same thing happened to me once when I was working on Windows, but I wasn't sure what caused it.)

File: frontend/src/pages/recruitment/Home.js [hunk 1]
```diff
@@ -89,6 +97,51 @@ const Home = () => {
       ))
     );
 
+  const reviewer_stats =
+    state.statsForReviewer == null ? (
+      <></>
+    ) : (
+      // Object.entries(state.statsForReviewer).map(([role, stats]) => (
```
Comment (justinyaodu) [inline frontend/src/pages/recruitment/Home.js:104]: Can we delete this commented-out code?

**Review by justinyaodu — APPROVED** [review]
Left a comment about removing some commented-out code. Looks good otherwise :+1:

**Review by hahsu — APPROVED** [review]
lgtm, thank you!

**Pull request merged**